attendance/views.py

```diff
@@ -58,15 +70,16 @@ def check_today_attendance(self, request):
         if attendance:
             serializer = AttendanceSerializer(attendance, many=True, context=serializer_context)
             return Response(serializer.data[0], status=status.HTTP_200_OK)
-        return JsonResponse({'detail': 'You did not check-in today'}, status=status.HTTP_404_NOT_FOUND)
+        return JsonResponse({'detail': 'You did not check-in today'}, status=status.HTTP_204_NO_CONTENT)
```

Comment:
404 is the correct error

Reply:
yes but if on employee login if employee first came to dashboard on new day it gives 404 error as he did not check in for today that's why I change it to 204 error.